rules/python/lang/cookies.yml

```diff
@@ -0,0 +1,45 @@
+imports:
+  - python_shared_lang_datatype
+patterns:
+  - pattern: $<COOKIE_INIT>[$<_>] = $<DATA_TYPE>
```

Comment:
^ Following discussion, this is somewhat expected and also, we're fine with this strictness being required here (specifying something has to be in the `[]`), since the new engine will also be strict in this way 👍